addons/mrp/models/mrp_production.py

```diff
@@ -484,6 +524,14 @@ def write(self, vals):
             moves.write({
                 'date_expected': fields.Datetime.to_datetime(vals['date_planned_start']),
             })
+
+        if 'date_deadline' in vals or 'date_planned_start' in vals or 'date_planned_finished' in vals:
```

Comment:
It really look like https://github.com/odoo/odoo/blob/13.0/addons/stock/models/stock_move.py#L464
It should probably be adapted to log on MO also.